frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunDetails.tsx

```diff
@@ -40,7 +42,14 @@ export const PipelineRunDetails: React.FC<PipelineRunDetailsProps> = ({ obj: pip
           <dl>
             <dt>Status</dt>
             <dd>
-              <Status status={pipelineRunFilterReducer(pipelineRun)} />
+              {pipelineRunStatus !== '-' ? (
+                <StatusIconAndText
+                  title={pipelineRunStatus}
+                  icon={<StatusIcon status={pipelineRunStatus} />}
+                />
+              ) : (
+                '-'
+              )}
```

Comment:
```suggestion
                <StatusIconAndText
                  title={pipelineRunStatus}
                  icon={<StatusIcon status={pipelineRunStatus} />}
                />
```
Making this much cleaner.